### ISSUE TITLE
better class structure on interface buttons

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#969]
they should have unique, logical class names so we can target them easily with voice and keyboard shortcuts as well as unique styling / other triggerable events. Right now it's difficult to accurately target the accessibility button to be clicked for example.
